utils/colors.ts

```diff
@@ -1,4 +1,13 @@
-const colors = {
+interface Colors{
+	"primary":string,
+	"secondary":string,
+	"text":string,
+	"invert":string,
+}
+
+type Themes= "bright" | "dark";
```

Comment:
Third time declaring the same type...



...




...




![image](https://github.com/zoop-de-loop/ask.io/assets/98516489/c2642dea-8d11-4aa9-991e-3458928abd0f)

Reply:
noted